### PR TITLE
Add Transcode Video doc

### PR DESCRIPTION
**Pull request by leszko**
## Description

Add doc for Transcode API

I've added it as 3 pages (as it was originally created by Yondon):
- Transcode Video
- Transcode Video Storj
- Transcode Video Web3 Storage

But we can consider having 1 page with subpages.

### CODE REVIEW

File: pages/reference/api/transcode-video-storj.en-US.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+There are [code examples](#code-examples) provided for reference.
+
+## Storj S3 Credentials
+
+This step assumes that you have already created an account at [https://www.storj.io/](https://www.storj.io/).
+
+Follow the [Storj guide for generating S3 credentials](https://docs.storj.io/dcs/api-reference/s3-compatible-gateway#I7p7Q) using either the web interface or `uplink`. Make sure that the credentials have the proper read/write permissions for the bucket that your video files will be read from and that transcoded videos will be written to. The access key and secret key of the credentials will be used in the next step.
+
+## Upload Video to Storj
```
Comment (hthillman) [inline pages/reference/api/transcode-video-storj.en-US.mdx:17]: can we link to Storj's upload docs here?

File: pages/reference/api/transcode-video-storj.en-US.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+Make sure you have a video file uploaded to your Storj bucket. You must upload the video file using one of the following methods:
+
+- Via the Storj web interface
+- Via a S3 API client
+    - [AWS has S3 API client implementations](https://aws.amazon.com/developer/tools/) in different programming languages
+    - The [code examples](#code-examples) illustrate how to upload a video file to Storj using a S3 API client
+
+Note: At the moment, you can *not* use `uplink` to upload the video file to Storj because `uplink` does not support generating the required S3 object metadata.
```
Comment (hthillman) [inline pages/reference/api/transcode-video-storj.en-US.mdx:26]: Is this inconsistent with the comment below re: uplink? we say you can't use uplink for upload but then describe how you can use it for playback?

File: pages/reference/api/transcode-video-storj.en-US.mdx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+   ]
+}'
+```
+
+- `$API_KEY` should be your Livepeer Studio API key
+- `$ACCESS_KEY_ID` should be the access key and `$SECRET_ACCESS_KEY` should be the secret key of the S3 credentials you generated in the previous step
+- `input.bucket` should be the name of your Storj bucket
+- `input.path` should be the path that your video file can be found in the Storj bucket
+- `outputs.hls.path` will be the path that the HLS playlist and mpegts segments will be found in when transcoding is complete
+- `profiles` is an optional parameter to specify the desired properties of transcoded video
```
Comment (hthillman) [inline pages/reference/api/transcode-video-storj.en-US.mdx:87]: Can you link to our profiles documentation?

**Review by hthillman — COMMENTED** [review]
left a couple of questions.

One other conceptual question:
all of these are currently in the API reference section, but the Storj and w3s docs are guides. should those be moved to the guides section? I'll add @adamsoffer as a reviewer since I know he's thinking about information architecture

File: pages/reference/api/transcode-video-storj.en-US.mdx (hunk 1 above)
Comment (leszko) [reply pages/reference/api/transcode-video-storj.en-US.mdx:17]: > can we link to Storj's upload docs here?

I don't think there is a good link to Storj documentation. We could like this one: https://docs.storj.io/dcs/getting-started/quickstart-uplink-cli/uploading-your-first-object.

But it's using `uplink` and there are two other (simpler) methods for file upload (web interface and AWS S3 CLI).

**Comment by leszko** [comment]
> left a couple of questions.
> 
> One other conceptual question: all of these are currently in the API reference section, but the Storj and w3s docs are guides. should those be moved to the guides section? I'll add @adamsoffer as a reviewer since I know he's thinking about information architecture

Moved to guides. PTAL

**Review by hthillman — APPROVED** [review]
LGTM 🚢

**Pull request merged**